pkg/validation/options.go

```diff
@@ -380,8 +380,17 @@ func parseJwtIssuers(issuers []string, msgs []string) ([]jwtIssuer, []string) {
 			msgs = append(msgs, fmt.Sprintf("invalid jwt verifier uri=audience spec: %s", jwtVerifier))
 			continue
 		}
-		uri, audience := components[0], strings.Join(components[1:], "=")
-		parsedIssuers = append(parsedIssuers, jwtIssuer{issuerURI: uri, audience: audience})
+		if len(components) > 3 {
+			msgs = append(msgs, fmt.Sprintf("invalid jwt verifier uri=jwksURI=audience spec: %s", jwtVerifier))
+			continue
+		}
+		if len(components) == 2 {
```

Comment:
So with the separator being variable length, this would break any legacy setups with an audience that has an embedded `=` -- which would be misidentified as a separator for a JWKS.

I think if we want to support this, it needs to be a separate flag with the format `issuer=jwks_uri`. We should immediately mark it as `Deprecated` since it will be removed in V8 for structured configs.

Reply:
What about a new bool flag to enable this behaviour? Like this it wont break legacy setups while keeping the code simple.

Reply:
Interesting thought 🤔 

Thinking it through, I think a distinct flag would be easier to maintain -- combo flags would make the validation package & unit tests around it more complex.